src/store/middleware/view.tsx

```diff
@@ -23,4 +23,16 @@ export const passViewMiddleware = (stateAPI: MiddlewareAPI<Dispatch, Application
         });
       });
   }
+
+  if (action.type === Action.EnableHotkeyNotifications) {
+    if (typeof window !== undefined) {
+      window.localStorage.setItem("scrumlr/hotkeyNotificationsEnabled", JSON.stringify(true));
```

Comment:
Please use `constants/storage.ts` and `utils/storage.ts` to access the locale storage

Reply:
Done ✅